pom.xml

```diff
@@ -361,7 +361,7 @@
             Properties for dependencies that solely relate to maven plugins go in the plugin section above.
             Properties for dependencies that solely relate to tests go in the test code section above.
          -->
-        <version.antlr>4.10</version.antlr>
+        <version.antlr>4.14.0</version.antlr>
```

Comment:
Thanks @nekop  for figuring out that we need to update ANTLR!

Please change to use Antlr version 4.13.0 as is used in https://github.com/hibernate/hibernate-orm/blob/6.4/settings.gradle#L67 and also is what the WFLY-18904 issue states as well.

Reply:
Woops, terrible typo. Fixed.

Reply:
Thank you for the change!